### PR TITLE
Add blog post about CRI-O's latest packaging efforts

### PR DESCRIPTION
**Pull request by saschagrunert**
Adding a blog post about CRI-O's packaging efforts.

PTAL @haircommander @xmudrii 

~The publishing date is just a placeholder for now, means:~
~/hold~

[Deploy preview](https://deploy-preview-43128--kubernetes-io-main-staging.netlify.app/blog/2023/10/10/cri-o-community-package-infrastructure/)

### CODE REVIEW

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (sftim) [inline content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:1]: Early feedback: site relative links please

ie `/blog/2023/08/31/legacy-package-repository-deprecation/` not `/blog/2023/08/31/legacy-package-repository-deprecation/`

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (haircommander) [inline content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:23]: style nit: I think you could drop `for sure` here

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (haircommander) [inline content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:149]: reconciled*

**Comment by haircommander** [comment]
thanks for putting this together! I have a couple of nits but generally LGTM

**Comment by xmudrii** [comment]
/assign
I'll make sure to take a look at this tomorrow. :)

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (dipesh-rawat) [inline content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:10]:
```suggestion
The Kubernetes community [recently announced](/blog/2023/08/31/legacy-package-repository-deprecation/)
```

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (dipesh-rawat) [inline content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:12]:
```suggestion
[introduced community-owned packages](/blog/2023/08/15/pkgs-k8s-io-introduction) powered by the
```

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (saschagrunert) [reply content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:23]: Fixed as suggested.

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (saschagrunert) [reply content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:10]: Fixed :+1:

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (saschagrunert) [reply content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:12]: Fixed :+1:

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (Arnett28) [reply content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:1]: Tanks

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (windsonsea) [inline content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:126]: Same concerns to split and change into a list if it is a sequencial procedure.

**Comment by sftim** [comment]
That publication date LGTM, so
/hold cancel

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (aj11anuj) [inline content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:11]:
```suggestion
that their legacy package repositories are frozen, and now they will move to 
```

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (aj11anuj) [inline content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:15]:
```suggestion
CRI-O has a long history of utilizing 
[OBS for their package builds](https://github.com/cri-o/cri-o/blob/e292f17/install.md#install-packaged-versions-of-cri-o), 
but all of the packaging efforts have been done manually so far.
```

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (aj11anuj) [inline content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:33]:
```suggestion
In the same way as the Kubernetes community, CRI-O provides `deb` and `rpm`
packages as part of a dedicated subproject in OBS, called 
[isv:kubernetes:addons:cri-o](https://build.opensuse.org/project/show/isv:kubernetes:addons:cri-o). 
This project acts as an umbrella and provides `stable` (for CRI-O tags) as well as 
`prerelease` (for CRI-O `releaes-1.y` and `main` branches) package builds.
```

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (aj11anuj) [inline content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:54]:
```suggestion
For example, If an end-user would like to install the latest available version
```

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (aj11anuj) [inline content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:61]:
```suggestion
For `rpm` based distributions, you can run the following commands as a `root` user 
to install CRI-O:
```

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (aj11anuj) [inline content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:120]:
```suggestion
For `deb` based distributions, you can run it as a `root` user:
```

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (aj11anuj) [inline content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:154]:
```suggestion
#### Start and enable CRI-O
```

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (aj11anuj) [inline content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:162]:
```suggestion
The Project's `prerelease:/main` prefix at the CRI-O's package path, can be replaced with 
`stable:/v1.28`, `stable:/v1.29`, `prerelease:/v1.28` or `prerelease:/v1.29`
if another package stream should be used.
```
Line 162: Clarify the meaning, did you wanted to say 
"If another package, stream should be used" or "If another stream package is used."

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (aj11anuj) [inline content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:165]:
```suggestion
Bootstrapping a cluster using `kubeadm` can be done by running `kubeadm init` command, 
which automatically detects that CRI-O is running in the background.
```

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (aj11anuj) [inline content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:174]:
```suggestion
Everything related to these packages lives in the new 
[CRI-O packaging repository](https://github.com/cri-o/packaging). 
It contains a [daily reconciliation](https://github.com/crio/packaging/actions/workflows/schedule.yml) 
GitHub action workflow, for all supported release branches as well as tags of
```

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (aj11anuj) [inline content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:186]:
```suggestion
The package build inputs will undergo daily reconciliation and will be supplied by 
CRI-O's [static binary bundles](https://console.cloud.google.com/storage/browser/cri-o/artifacts). 
These bundles are built and signed for each commit in the CRI-O CI, 
and contain everything CRI-O requires to run on a certain architecture. 
The static builds are reproducible, powered by [nixpkgs](https://github.com/NixOS/nixpkgs) 
and available only for `x86_64`, `aarch64` and `ppc64le` architecture.
```

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (aj11anuj) [inline content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:191]:
```suggestion
The CRI-O maintainers will be happy to listen to any feedback or suggestions on the new
packaging efforts! Thank you for reading this blog post, feel free to reach out
to the maintainers via the Kubernetes [Slack channel #crio](https://kubernetes.slack.com/messages/CAZH62UR1)
or create an issue in the [packaging repository](https://github.com/cri-o/packaging/issues).
```

**Review by aj11anuj — CHANGES_REQUESTED** [review]
Here are some suggestions from me, for maintaining reading fluency of blog.

**Comment by saschagrunert** [comment]
Thank you for the review @aj11anuj, I applied the suggestions! :pray:

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (aj11anuj) [inline content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:164]: Line 164: Clarification of this sentence is required, did you mean

`if another package, stream should be used` (in this case, you are missing a comma `,`)
or 
`if another stream package is used`

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (aj11anuj) [inline content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:181]:
```suggestion
packages is done with the help of the Kubernetes Release [Toolbox (krel)](https://github.com/kubernetes/release/blob/master/docs/krel/README.md),
```

**Review by aj11anuj — CHANGES_REQUESTED** [review]
.

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (saschagrunert) [reply content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:164]: The second one. :+1:

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (saschagrunert) [reply content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md]: Fixed :+1:

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (aj11anuj) [inline content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:164]:
```suggestion
if another stream package is used.
```

**Review by aj11anuj — CHANGES_REQUESTED** [review]
Commit 2nd sentence as a change, if you mean 2nd one.

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (xmudrii) [inline content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:11]: > and now they will move to

I would change this part because we already moved to the new infrastructure.

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (xmudrii) [inline content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:12]:
```suggestion
[introduced community-owned package repositories](/blog/2023/08/15/pkgs-k8s-io-introduction) powered by the
```

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (xmudrii) [inline content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:13]:
```suggestion
[OpenBuildService (OBS)](https://build.opensuse.org/project/subprojects/isv:kubernetes).
```

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (xmudrii) [inline content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:27]: > as well as the corresponding release branches.

This is slightly confusing to me.

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (xmudrii) [inline content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:33]:
```suggestion
[`isv:kubernetes:addons:cri-o`](https://build.opensuse.org/project/show/isv:kubernetes:addons:cri-o).
```

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (xmudrii) [inline content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:76]: We can potentially drop this because you can use CRI-O repos without Kubernetes repos. Instead, we can link to the official installation docs.

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (xmudrii) [inline content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:122]:
```suggestion
For `deb` based distributions, you can run the following commands as a `root` user:
```

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (xmudrii) [inline content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:138]: We can potentially drop this because you can use CRI-O repos without Kubernetes repos. Instead, we can link to the official installation docs.

**Review by xmudrii — CHANGES_REQUESTED** [review]
Very happy to see CRI-O moving towards pkgs.k8s.io! Some nits, but otherwise, looks great!

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (saschagrunert) [reply content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:76]: We require `kubernetes-cni` in the CRI-O spec.

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (saschagrunert) [reply content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md]: Fixed

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (saschagrunert) [reply content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md]: Rephrased to mention the actual release branch.

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (xmudrii) [reply content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:76]: Can we add this as a note? Something like: "we need the kubernetes repo because CRI-O depends on the `kubernetes-cni` package hosted in that repo"

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (xmudrii) [inline content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:111]: I would drop this part and later on link to the official instructions. There are some other considerations that are not covered by this blog post, like binding package versions.

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (xmudrii) [inline content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:161]: Let's link this document here: https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (saschagrunert) [reply content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:133]: I'd be in favor of having the docs how to setup k8s with CRI-O in one place. We can also follow-up and update the kubeadm docs at some later point in time.

**Review by xmudrii — COMMENTED** [review]
Some more nits, otherwise LGTM

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (saschagrunert) [reply content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md]: Done :+1:

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (xmudrii) [reply content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:111]: Okay, that makes sense 👍

**Review by xmudrii — APPROVED** [review]
LGTM, thank you for doing this!

**Comment by saschagrunert** [comment]
@kubernetes/sig-docs-blog-owners @kubernetes/sig-docs-pr-reviews can we get this merged, looks like we reached general approval on the PR.

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (sftim) [inline content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:35]:
```suggestion
`prerelease` (for CRI-O `releases-1.y` and `main` branches) package builds.
```

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (sftim) [inline content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:164]: Better to link to a commit ref not a branch; commits are less likely to turn stale.

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (sftim) [inline content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:180]:
```suggestion
CRI-O's static binary bundles.
```

We shouldn't assume that readers have a Google login.

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (sftim) [inline content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:171]: I'd link to the source code of the action and use a commit ref not a branch.

File: content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md
Comment (sftim) [inline content/en/blog/_posts/2023-10-10-cri-o-community-package-infrastructure.md:176]: I'd link to a commit ref not a branch.

**Review by sftim — COMMENTED** [review]
OK to merge and to publish, but this _is_ likely to go stale.

**Comment by saschagrunert** [comment]
@sftim thanks, I'm following up on this in https://github.com/kubernetes/website/pull/43313

**Pull request merged**
